superset-frontend/plugins/plugin-chart-echarts/src/Timeseries/transformers.ts

```diff
@@ -84,6 +84,7 @@ export function transformSeries(
     thresholdValues?: number[];
     richTooltip?: boolean;
   },
+  rawSeriesB?: SeriesOption[],
```

Comment:
transformSeries means how to generate echart SeriesOption from form_data. we can set different attributes on the different series to resolve this issue.

```suggestion
   richTooltip?: boolean;
   seriesHash?: string;
  },
```

Reply:
I think it makes sense. However, I have a concern that since the label value is specified by colorScale function, there may be some problem if user wants to set the custom label colors. User may not know how to set the label color of the chart B in this case.

Reply:
> I think it makes sense. However, I have a concern that since the label color is specified by colorScale function, there may be some problem if user wants to set the custom label colors.

Good question! we need to apply a customize color panel before apply random string.